### PR TITLE
Relative imports

### PR DESCRIPTION
**Pull request by mbucknell**
Before making a pull request
----------------------------

- [X] Make sure all tests run
- [ ] Update the changelog appropriately (The changelog doesn't need to be updated for this internal code change)

Title
-----------
Use relative imports when importing a module within the same directory. The named aliases are used when importing a module from a different directory. This change affects the .spec.js files the most. 

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: assets/src/scripts/d3-rendering/tooltips.js [hunk 1]
```diff
@@ -7,7 +7,7 @@
  */
 
 // Required to initialize USWDS components after page load
-import components from '../../node_modules/uswds/src/js/components';
+import components from 'uswds/src/js/components';
```
Comment (abriggs-usgs) [inline assets/src/scripts/d3-rendering/tooltips.js:10]: Is this an 'alias-plugin' thing, or just a standard Rollup thing?

File: assets/src/scripts/d3-rendering/tooltips.spec.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import {select} from 'd3-selection';
-import {appendTooltip} from 'ui/tooltips';
 
+import {appendTooltip} from './tooltips';
 
-describe('tooltips', () => {
+
+describe('d3-rendering/tooltips', () => {
```
Comment (abriggs-usgs) [inline assets/src/scripts/d3-rendering/tooltips.spec.js:6]: Is just to tell us which tooltips are being tested, or is there greater significance?

File: assets/src/scripts/monitoring-location/components/daily-value-hydrograph/selectors/legend-data.spec.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
-import {lineMarker, rectangleMarker, textOnlyMarker} from 'd3render//markers';
```
Comment (abriggs-usgs) [inline assets/src/scripts/monitoring-location/components/daily-value-hydrograph/selectors/legend-data.spec.js:1]: How did this still work??

**Review by abriggs-usgs — APPROVED** [review]
Thanks for cleaning that up. Much improved.

File: assets/src/scripts/d3-rendering/tooltips.js (hunk 1 above)
Comment (mbucknell) [reply assets/src/scripts/d3-rendering/tooltips.js:10]: Just standard rollup. I'm not sure why I didn't try it this way before.

File: assets/src/scripts/d3-rendering/tooltips.spec.js (hunk 2 above)
Comment (mbucknell) [reply assets/src/scripts/d3-rendering/tooltips.spec.js:6]: Yeah. I have been trying to change these so it's easier in the test output to figure out where the error occurred. In this particular case we have tooltips in d3-rendering and the then tooltip in hydrograph and dv-hydrograph.

File: assets/src/scripts/monitoring-location/components/daily-value-hydrograph/selectors/legend-data.spec.js (hunk 3 above)
Comment (mbucknell) [reply assets/src/scripts/monitoring-location/components/daily-value-hydrograph/selectors/legend-data.spec.js:1]: You mean with the double slashes? I think it just ignored it. Some things seem to be picky about that put clearly the rollup code is not .

**Pull request merged**